genie-demo/src/main/docker/client/example/init_demo.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python2.7
+
+# Copyright 2016 Netflix, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import yaml
+from pygenie.conf import GenieConf
+from pygenie.client import Genie
+
+logging.basicConfig(level=logging.ERROR)
+
+LOGGER = logging.getLogger(__name__)
+
+
+def load_yaml(yaml_file):
+    with open(yaml_file) as _file:
+        return yaml.load(_file)
+
+genie_conf = GenieConf()
+genie_conf.genie.url = "http://genie:8080"
+
+genie = Genie(genie_conf)
+
+hadoop_application = load_yaml("applications/hadoop271.yml")
+hadoop_application_id = genie.create_application(hadoop_application)
+LOGGER.info("Created hadoop application with id = [%s]" % hadoop_application_id)
```

Comment:
You're setting the `LOGGER` to `ERROR` so I don't think you'll see these lines. Also #42, #46, etc.